apps/openmw/mwgui/container.cpp

```diff
@@ -82,7 +82,7 @@ namespace MWGui
             dialog->eventOkClicked.clear();
             dialog->eventOkClicked += MyGUI::newDelegate(this, &ContainerWindow::dragItem);
         }
-        else
+        else if (mModel)
```

Comment:
That is an odd place to fix it. Had to look up the source to understand what is going on. IMHO the check mModel would be better placed inside of dragItem.

Reply:
Moved.
Now I check if model exists inside dragItem() and dragItem() calls.